src/coreclr/inc/regdisp.h

```diff
@@ -397,7 +425,24 @@ inline void FillContextPointers(PT_KNONVOLATILE_CONTEXT_POINTERS pCtxPtrs, PT_CO
     {
         *(&pCtxPtrs->Edi + i) = (&pCtx->Edi + i);
     }
-#else // TARGET_X86
+#elif defined(TARGET_RISCV64) // TARGET_X86
+    // *(&pCtxPtrs->S0) = &pCtx->S0;
```

Comment:
is this commented out on purpose?

Reply:
Actually, I did on purpose. In RISC-V, S0 is used as FP. (FP is handled in line 442.) I want to keep it as commented.
However, I think it can be removed.